app/src/main/java/de/westnordost/streetcomplete/map/MapFragment.kt

```diff
@@ -218,7 +221,12 @@ open class MapFragment : Fragment(),
     protected open fun getSceneFilePath(): String {
         val currentNightMode = resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK
         val isNightMode = currentNightMode == Configuration.UI_MODE_NIGHT_YES
-        val scene = if (isNightMode) "scene-dark.yaml" else "scene-light.yaml"
+        val isAerialView = prefsShared.getString(Prefs.THEME_BACKGROUND, "MAP") == "AERIAL_ESRI"
+
+        val scene = if (isAerialView) "scene-satellite.yaml" else {
+            if (isNightMode) "scene-dark.yaml" else "scene-light.yaml"
+        }
```

Comment:
That would make it a bit easier to read, I guess:

```suggestion
        val scene = when {
            isAerialView -> "scene-satellite.yaml"
            isNightMode -> "scene-dark.yaml"
            else -> "scene-light.yaml"
        }
```

Reply:
Looks nice - but I will need to check is it working. What if both isAerialView and isNightMode are true?

Yes, it works: https://kotlinlang.org/docs/control-flow.html#when-expression